src/lib/helpers.js

```diff
@@ -0,0 +1,10 @@
+function getUnhandledProps(ComponentProps, props) {
+  const handledProps = Object.keys(ComponentProps);
+  return Object.keys(props).reduce((acc, prop) => {
+    if (prop === 'childkey') return acc;
```

Comment:
Why `childKey` here? I'm not quite sure I understand it

Reply:
You're right, this shouldn't be there. Just removed it.